test/NumberTheory/QuadFormAndIsom.jl

```diff
@@ -1,5 +1,5 @@
-using Test
-using Oscar
+include(joinpath(Oscar.oscardir, "test", "Serialization", "setup_tests.jl"))
+include(joinpath(Oscar.oscardir, "test", "Serialization", "upgrades", "setup_tests.jl"))
```

Comment:
you shouldn't need these lines here.

Reply:
> you shouldn't need these lines here. Does the error have to do with the upgrade script? whats the error exactly?
The errors I encountered are "`test_save_load_roundtrip` not defined" and same with "`test_1_4_0_upgrade` is not defined".
Following Lars' suggestion, I have moved the "test save roundtrip" tests inside the `Serialization/QuadForm.jl` test file, and for the upgrade test, I have removed `ZZLatWithIsom` from the exclude list in file `Serialization/upgrades/runtests` (last line). See the upcoming commit

Reply:
Yeah, looks good like this. Let's wait for CI to agree